deepspeed/launcher/runner.py

```diff
@@ -193,6 +193,8 @@ def parse_args(args=None):
                         "numbers and range. i.e. 1,3-5,7 => [1,3,4,5,7].  When not "
                         "specified, all cores on system would be used rank binding")
 
+    parser.add_argument("--ssh_port", type=int, default=22, help="SSH port to use for remote connections")
```

Comment:
I think the later checks for `if args.ssh_port is not None` will never trigger since the default here is `22`. Can we set the default to `None`?